docs/sequence-diagrams/store.puml

```diff
@@ -0,0 +1,4 @@
+@startuml
+thanos_store -> cloud_storage: request metadata
```

Comment:
Definitely too small amount of details, again as rule let's drop it and focus on this in details in another PRs .